app/routes/post_login.js

```diff
@@ -38,15 +38,36 @@ export default Ember.Route.extend(preloadDataMixin, {
         new AjaxPromise(`/orders/${draftOrder.id}`, "PUT", this.get('session.authToken'), { order: orderParams })
           .then(data => {
             this.get("store").pushPayload(data);
+            this.redirectToTransitionOrDetails();
           });
```

Comment:
I've been seeing a lot of uncaught promises and async calls in Ember. Should we not add a .catch() hook to the promises, how are the errors handled ? 
Unless there is an "ember way" of handling things, I think promises should be caught.